Automattic/vip-go-mu-plugins
# Enable Jetpack CP reconnection filter by default

## Description

We've seen some new sites that are not connected automatically to Jetpack because of a check in Connection Pilot. Since it's the default behavior, we want to default to connect.

This PR changes how the filter that checks whether the site should be reconnected or not works. The new default behavior is trying to reconnect.

## Changelog Description

### Enable Jetpack CP reconnection filter by default

Jetpack Connection Pilot would not connect automatically on some new sites.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Create a new site on a sandbox
3. Jetpack should automatically be connected.
